meta-balena-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -70,3 +70,8 @@ FILES_${PN} += "/usr/src/*"
 # Tools inside the headers package are slightly special.
 # Skip some QA checks. We are interested in the arch check only.
 INSANE_SKIP_${PN} = "file-rdeps ldflags staticdev already-stripped"
+
+# gen_mod_headers uses host tools while running make modules_prepare
+# gcc on the host might not support the -fmacro-prefix-map option which
+# is quite recent and introduced in Yocto since warrior. Remove it for this recipe
+DEBUG_PREFIX_MAP_remove = "-fmacro-prefix-map=${WORKDIR}=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR}"
```

Comment:
I'm not sure I understand. Aren't the host tools part of upstream? And if they are, don't they have the needed support accordingly?

Reply:
@agherzan this one is nasty. by host tools I meant build host! yea. native ubuntu gcc5 was being used.

tbh. I don't want to invest more effort into this recipe as we want to nuke it.

Reply:
Right. I'm OK with that. Was just curious to know what was with this.